library/X509/Widget/JobDetails.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/* Icinga Web 2 X.509 Module | (c) 2023 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\X509\Widget;
+
+use Icinga\Module\X509\Model\X509JobRun;
+use ipl\Html\Table;
+use ipl\I18n\Translation;
+use ipl\Orm\Query;
+use ipl\Web\Widget\EmptyState;
+
+class JobDetails extends Table
+{
+    use Translation;
+
+    protected $defaultAttributes = ['class' => 'common-table'];
+
+    /** @var Query */
+    protected $runs;
+
+    public function __construct(Query $runs)
+    {
+        $this->runs = $runs;
+    }
+
+    protected function assemble(): void
+    {
+        /** @var X509JobRun $run */
+        foreach ($this->runs as $run) {
+            $row = static::tr();
+            $row->addHtml(
+                static::td($run->job->name),
+                static::td($run->schedule->name ?: $this->translate('N/A')),
+                static::td((string) $run->total_targets),
+                static::td((string) $run->finished_targets),
+                static::td($run->start_time->format('Y-m-d H:i')),
+                static::td($run->end_time ? $run->end_time->format('Y-m-d H:i') : 'N/A')
+            );
+
+            $this->addHtml($row);
+        }
+
+        if ($this->isEmpty()) {
+            $this->addHtml(new EmptyState($this->translate('Job never run.')));
```

Comment:
`$this` is a `table`, the empty state (bar) a `div`, and a `div` isn't valid inside a `table`. That's why ipl-web transforms an item list/table to a `div` if it's adding the bar, you should do the same. (And everywhere else where you're using this)